WordPress/src/main/res/layout/publicize_list_fragment.xml

```diff
@@ -51,7 +51,7 @@
                 android:textColor="@color/grey_dark"
                 android:textSize="@dimen/text_sz_medium" />
 
-            <org.wordpress.android.widgets.WPButton
+            <Button
```

Comment:
I'm not sure how that would affect the button's size, but that was a change you made here: 4590c8f08b7c37eaa942041ede9b3d9fce4dde36